src/modules/colorPicker/ColorPickerUI/ViewModels/ColorEditorViewModel.cs

```diff
@@ -324,9 +324,15 @@ private void SetupAllColorRepresentations()
             _allColorRepresentations.Add(
                 new ColorFormatModel()
                 {
-                    FormatName = "Decimal",
+                    FormatName = "Decimal BGR",
                     Convert = (Color color) => ColorRepresentationHelper.GetStringRepresentationFromMediaColor(color, ColorRepresentationType.DecimalValue),
                 });
+            _allColorRepresentations.Add(
+                new ColorFormatModel()
+                {
+                    FormatName = "Decimal RGB",
```

Comment:
Unfortunately the UI doesn't seem to be able to show all this and shows just "Decimal"
![image](https://user-images.githubusercontent.com/26118718/170075786-369c9ed1-46b6-468f-a88b-b8cdfbd3ad64.png)

Might it make sense to call this "Deci RGB" for the time being?